sdk/loadtestservice/load-testing-rest/samples-dev/sample.ts

```diff
@@ -0,0 +1,63 @@
+import AzureLoadTesting, { AzureLoadTestingClient } from "@azure-rest/load-testing";
+import { DefaultAzureCredential } from "@azure/identity";
+import dotenv from "dotenv";
+import { createReadStream } from "fs";
+
+const readStream = createReadStream("./sample.jmx");
+dotenv.config({ path: "../../sample.env" });
+
+async function main() {
+  const Endpoint = process.env["LOADTESTSERVICE_ENDPOINT"] || "";
+  const DISPLAY_NAME = "some-load-test";
+  const SUBSCRIPTION_ID = process.env["SUBSCRIPTION_ID"] || "";
+  const Client: AzureLoadTestingClient = AzureLoadTesting(Endpoint, new DefaultAzureCredential());
+
+  await Client.path("/loadtests/{testId}", "abc").patch({
+    contentType: "application/merge-patch+json",
+    body: {
+      displayName: "mrinal-test",
+      description: "JS SDK",
+      loadTestConfig: {
+        engineInstances: 1,
+        splitAllCSVs: false,
+      },
+    },
+  });
+
+  await Client.path("/loadtests/{testId}/files/{fileId}", "abc", "xyz12365").put({
```

Comment:
Same comment here, I'd recommend also creating variables instead of inlining the id values to make sure they stand up as things users need to take care of